global.json

```diff
@@ -1,6 +1,6 @@
 {
   "tools": {
-    "dotnet": "9.0.100"
+    "dotnet": "9.0.102-servicing.24577.25"
```

Comment:
Is this why the feed is required above? Could you just use 9.0.101 that just came out?

Reply:
Yap, kinda of, right now the setup I have it needs to provision a new dotnet, if the global dotnet matches my script fails, I need to fix that. thats why I bumped this.

Reply:
If this is main, was the policy to only use stable/released bits? Like 9.0.101?

Reply:
This is only used by our internal setup on dnceng and builds with arcade. For a developer suing cake shouldn't t make a difference. the version for the sdk on the version props is still 9.0.100

Reply:
fixed with 101